ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -359,7 +359,7 @@ public int reactTagForTouch(float touchX, float touchY) {
         for (int i = 0; i < spans.length; i++) {
           int spanStart = spannedText.getSpanStart(spans[i]);
           int spanEnd = spannedText.getSpanEnd(spans[i]);
-          if (spanEnd > index && (spanEnd - spanStart) <= targetSpanTextLength) {
+          if (spanEnd >= index && (spanEnd - spanStart) <= targetSpanTextLength) {
```

Comment:
the main question is why it did work on non-nested Text. 
I need to find the reason spanEnd reaches the index value on the last characted when the Text is nested in another Text

Reply:
The issue was not experienced on non-nested text as normal Text will not generate Spans. Spans are generated by nested Text react tags